databricks_cli/sdk/service.py

```diff
@@ -706,3 +706,45 @@ def get_groups_for_principal(self, user_name=None, group_name=None, headers=None
             _data['group_name'] = group_name
         return self.client.perform_query('GET', '/groups/list-parents', data=_data, headers=headers)
 
+
+class DeltaPipelinesService(object):
```

Comment:
Is this hand written? We should avoid doing that since this file is autogenerated and these changes will get clobbered next time someone runs the autogenerator.

Reply:
It is autogenerated, but the autogeneration PR is not merged into Universe yet. I've created a JIRA task to ensure that when the Pipelines branch on the CLI gets merged with master, so does the PR that does the autogeneration get merged into Universe. 
The reason to hold off on the merging into Universe is we don't want someone trying to add their component and accidentally adding the Pipelines service as part of that.